app/main/index.js

```diff
@@ -129,6 +139,20 @@ function createMainWindow() {
 // Decrease load on GPU (experimental)
 app.disableHardwareAcceleration();
 
+if (process.platform === 'win32') {
+	execPath = 'C:\\Program Files\\Zulip\\Zulip.exe';
```

Comment:
what if the user installs Zulip at a different path?

Reply:
> (I've hard-coded the path to the Windows executable here, please let me know if there's a better way to handle this)

That's what I've asked about here, I didn't look into getting the install path yet.

Reply:
Also, Vipul, it'd be great if you could check out this PR locally and test it on your system. I'm guessing Linux might have some issues with this sort of feature.

Reply:
Sure, will check till evening :)

Reply:
How do i test this on Ubuntu? Simply prefixing zulip:// to normal URLs doesn't seem to be working. Anything i am missing here?

Reply:
Don't think so - this method works without any additional work on macOS and Windows. 
Maybe the desktop file needs to be configured properly. Let me check. I'll bother you on czo with requests for testing later tonight. :P

Reply:
no problem. i'm here.